src/components/floating-emojis/FloatingEmoji.js

```diff
@@ -24,73 +26,82 @@ const FloatingEmoji = ({
   top,
   wiggleFactor,
 }) => {
-  const { opacity, rotate, scale, translateX, translateY } = useMemoOne(() => {
-    const animation = timing({ duration, easing: EasingNode.elastic() });
-    const progress = interpolate(animation, {
-      inputRange: [0, 1],
-      outputRange: [0, distance],
+  const animation = useSharedValue(0);
+
+  useLayoutEffect(() => {
+    animation.value = withTiming(1, {
+      duration,
+      easing: Easing.linear,
     });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
I think I would do something like:
```suggestion
  animation.value = useMemo(() => withTiming(1, {
      duration,
      easing: Easing.linear,
    }),
  []);
```

Reply:
maybe `useMemoOne`?